PrebidMobile/API1.0Demo/src/main/java/org/prebid/mobile/app/DemoActivity.java

```diff
@@ -63,7 +67,99 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             createMoPubBanner(intent.getStringExtra(Constants.AD_SIZE_NAME));
         } else if ("MoPub".equals(intent.getStringExtra(Constants.AD_SERVER_NAME)) && "Interstitial".equals(intent.getStringExtra(Constants.AD_TYPE_NAME))) {
             createMoPubInterstitial();
+        } else if ("DFP".equals(intent.getStringExtra(Constants.AD_SERVER_NAME)) && "Native".equals(intent.getStringExtra(Constants.AD_TYPE_NAME))) {
+            createDFPNative();
+        } else if ("MoPub".equals(intent.getStringExtra(Constants.AD_SERVER_NAME)) && "Native".equals(intent.getStringExtra(Constants.AD_TYPE_NAME))) {
+            createMoPubNative();
+        }
+    }
+
+    void createMoPubNative() {
```

Comment:
I see an PBS error `Invalid request format: No stored imp found for id: 25e17008-5081-4676-94d5-923ced4359d3`

Reply:
You are right. I used Rubicon host. Currently I see successful response from PBS but MoPub response is empty

``` JSON
{
"ad-responses": [{
    "content": "",
    "metadata": {
        "x-adtype": "clear",
        "x-backfill": "clear",
        "x-refreshtime": 999999
        }
    }],
    "x-request-id": "edc0d651ef8747e2bcfdad8c37f327cb_0090428a006172cd"
}

```